kotsadm/pkg/apiserver/server.go

```diff
@@ -44,6 +48,8 @@ func Start() {
 		panic(err)
 	}
 
+	detectInstanceID()
```

Comment:
This should follow the pattern used elsewhere in the code: return the error and log it here at the caller's level